cognitojwt/jwt_async.py

```diff
@@ -33,7 +33,13 @@
     return jwk.construct(key)
 
 
-async def decode_async(token: str, region: str, userpool_id: str, app_client_id: str=None, testmode=False) -> dict:
+async def decode_async(
+        token: str,
+        region: str,
+        userpool_id: str,
+        app_client_id: Optional[str] = None,
```

Comment:
Optional[] is redundant if it has a default value

Reply:
fixed